packages/terra-card/src/CardBody.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import styles from './CardBody.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Child Nodes
+   */
+  children: PropTypes.node,
+  /**
+   * Provides themeable padding vertical
+   */
+  hasPaddingVertical: PropTypes.bool,
+  /**
+   * Provides themeable padding horizontal
+   */
+  hasPaddingHorizontal: PropTypes.bool,
+};
+
+const defaultProps = {
+  hasPaddingHorizontal: false,
+  hasPaddingVertical: false,
```

Comment:
Default these to both to true. The reason to use the body is for padding so we always want this defaulted